timber-lint/src/main/java/timber/lint/TimberIssueRegistry.kt

```diff
@@ -0,0 +1,16 @@
+@file:Suppress("UnstableApiUsage")
+
+package timber.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.detector.api.CURRENT_API
+import com.android.tools.lint.detector.api.Issue
+
+class TimberIssueRegistry : IssueRegistry() {
+  override val api = CURRENT_API
+  override val minApi: Int = CURRENT_API
```

Comment:
This is new. Not sure we want this.

Reply:
This is consistent with how androidx writes and publishes lint rules. 
Here is an example: https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:work/workmanager-lint/src/main/java/androidx/work/lint/WorkManagerIssueRegistry.kt

Reply:
Tor said you should make it as low as you test: https://github.com/JakeWharton/timber/issues/329#issuecomment-424534958

Reply:
Done.